### PR TITLE
feat: improved plugin #1631

### LINKED ISSUE #1631: improve Plugin & Plugin Editor & Meta Raw Data Editor module
Here are some minor issues on the Plugin & Plugin Editor & Meta Raw Data Editor page we could resolve.

## Plugin TODO

- [ ] Search area, using Table's own, using fuzzy search.
- [ ] page title not internationalized: configure global plugins/Configure Global Plugin.
- [ ] Add icon for Plugin Marketplace plugins.
- [ ] list page top title, add description: list of enabled plugins.
- [ ] list page, edit button in action area modified to: `配置/Configure`.

## Plugin Editor

- [ ] Some Auth plugins do not need to be configured, in the Data Editor, to disable the input of content.
- [ ] The button is not internationalized.
- [ ] Document button fits too close to JSON button.
- [ ] JSONSchema checks for error messages, which can be divided into: error message + view details.
- [ ] CodeMirror code error highlighting.

## Meta Raw data editor (Route List page)
- [ ] Button not internationalized.
- [ ] layout needs to be adjusted, with additional scrolling axes.

### PR DESCRIPTION
**Pull request by iamayushdas**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

  ## Plugin TODO
 * [ ]  Search area, using Table's own, using fuzzy search.
 * [x]  page title not internationalized: configure global plugins/Configure Global Plugin.
 * [ ]  Add icon for Plugin Marketplace plugins.
 * [x]  list page top title, add description: list of enabled plugins.
 * [x]  list page, edit button in action area modified to: `配置/Configure`.
 
 ## Plugin Editor
 * [ ]  Some Auth plugins do not need to be configured, in the Data Editor, to disable the input of content.
 * [x]  The button is not internationalized.
 * [ ]  Document button fits too close to JSON button.
 * [ ]  JSONSchema checks for error messages, which can be divided into: error message + view details.
 * [ ]  CodeMirror code error highlighting.
 
 ## Meta Raw data editor (Route List page)
 * [ ]  Button not internationalized.
 * [ ]  layout needs to be adjusted, with additional scrolling axes.

Extras: Dashboard title also internationalised

**Related issues**

fix/resolve #1631 

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

screencapture-172-20-10-3-8000-plugin-list-2021-03-26-08_32_08

screencapture-172-20-10-3-8000-plugin-list-2021-03-26-08_19_00

screencapture-172-20-10-3-8000-plugin-list-2021-03-26-08_33_35

### CODE REVIEW

File: web/src/locales/zh-CN/component.ts [hunk 1]
```diff
@@ -21,19 +21,24 @@ export default {
   'component.global.search': '查询',
   'component.global.reset': '重置',
   'component.global.confirm': '确认',
+  'component.global.format': '格式',
+  'component.global.document': '文档',
+  'component.global.enable': '使能够',
```
Comment (juzhiyuan) [inline web/src/locales/zh-CN/component.ts:26]:
```suggestion
  'component.global.enable': '启用',
```

File: web/src/locales/zh-CN/component.ts [hunk 2]
```diff
@@ -21,19 +21,24 @@ export default {
   'component.global.search': '查询',
   'component.global.reset': '重置',
   'component.global.confirm': '确认',
+  'component.global.format': '格式',
```
Comment (juzhiyuan) [inline web/src/locales/zh-CN/component.ts:24]:
```suggestion
  'component.global.format': '格式化',
```

File: web/src/locales/zh-CN/component.ts [hunk 3]
```diff
@@ -21,19 +21,24 @@ export default {
...
+  'component.global.document': '文档',
+  'component.global.enable': '使能够',
+  'component.global.scope': '范围',
+  'component.global.data.editor': '资料编辑器',
```
Comment (juzhiyuan) [inline web/src/locales/zh-CN/component.ts:28]:
```suggestion
  'component.global.data.editor': '原始内容编辑器',
```

File: web/src/locales/zh-CN/component.ts [hunk 4]
```diff
@@ -21,19 +21,24 @@ export default {
   'component.global.search': '查询',
   'component.global.reset': '重置',
   'component.global.confirm': '确认',
+  'component.global.format': '格式',
+  'component.global.document': '文档',
+  'component.global.enable': '使能够',
+  'component.global.scope': '范围',
```
Comment (juzhiyuan) [inline web/src/locales/zh-CN/component.ts:27]:
```suggestion
  'component.global.scope': '作用域',
```

File: web/src/pages/Plugin/locales/zh-CN.ts [hunk 5]
```diff
@@ -16,5 +16,5 @@
  */
 export default {
   'page.plugin.drawer.popconfirm.title.delete': '确定删除该插件吗？',
-  'page.plugin.list': '插件列表'
+  'page.plugin.list': '插件列表：已启用插件的列表'
```
Comment (juzhiyuan) [inline web/src/pages/Plugin/locales/zh-CN.ts:19]: Hi, as for the subtitle, how about using the `extra` prop like here[1]?

[1] https://procomponents.ant.design/en-US/components/page-container

**Review by juzhiyuan — COMMENTED** [review]
Wow! Great you are, and we could improve some translations for a better view!

File: web/src/pages/Plugin/locales/zh-CN.ts (hunk 5 above)
Comment (iamayushdas) [reply web/src/pages/Plugin/locales/zh-CN.ts:19]: okay i have closed this and will try to fix this in next PR

Comment (iamayushdas) [reply web/src/pages/Plugin/locales/zh-CN.ts:19]: screencapture-172-20-10-3-8000-plugin-list-2021-03-26-08_19_00

is it cool now?

**Comment by iamayushdas** [comment]
@juzhiyuan can you retrigger CI and review this now?

**Pull request closed without merging**